commerce-notification-service/src/main/java/com/liferay/commerce/notification/internal/upgrade/v2_1_0/CommerceNotificationQueueEntryUpgradeProcess.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.notification.internal.upgrade.v2_1_0;
+
+import com.liferay.commerce.notification.model.impl.CommerceNotificationQueueEntryImpl;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Alessio Antonio Rendina
+ */
+public class CommerceNotificationQueueEntryUpgradeProcess
+	extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		_addColumn(
+			CommerceNotificationQueueEntryImpl.class,
+			CommerceNotificationQueueEntryImpl.TABLE_NAME, "classNameId",
+			"LONG");
+		_addColumn(
+			CommerceNotificationQueueEntryImpl.class,
+			CommerceNotificationQueueEntryImpl.TABLE_NAME, "classPK", "LONG");
+	}
+
+	private void _addColumn(
```

Comment:
@ethanbustad I do see many of this method that is just a copy and past, What do you think to create a Base Class or a Util ?  Can you take care?

Reply:
Yeah, that's a great idea! I'll try to do that soon.